.github/workflows/style-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Style Checks
+on:
+  push:
+
+env:
+  working-directory: .
```

Comment:
working directory is not necessary when it is in the root of the project

Reply:
Fixed in e495f068835648b821bef8e10eeb23be6eb3408a